kivy/properties.pyx

```diff
@@ -1386,27 +1386,47 @@ cdef class AliasProperty(Property):
     If you don't find a Property class that fits to your needs, you can make
     your own by creating custom Python getter and setter methods.
 
-    Example from kivy/uix/widget.py::
+    Example from kivy/uix/widget.py where x` and `width` are instances of
+    `NumericProperty`::
```

Comment:
Use this:

    :class:`NumericProperty`

to get the reference for `NumericProperty` in the docs.